underfs/obs/src/main/java/alluxio/underfs/obs/OBSOutputStream.java

```diff
@@ -158,5 +163,6 @@ public void close() throws IOException {
       LOG.error("Failed to upload {}. Temporary file @ {}", mKey, mFile.getPath());
       throw new IOException(e);
     }
+    mFile.delete();
```

Comment:
why is this called twice (inside the try and outside)?

Reply:
Removed, thanks